README.md

```diff
@@ -36,17 +36,15 @@ We focus to solve the web development experience for backend developers and newb
 + Web developers need better performance in their web APPs
   + They can use some sepcified component from our libraray
 
-
 ## Personal connection
 
 > What is your personal connection to the service & the user group you are planning to reach with your product? *
 
+We currently have little idea about this.
 
-## Work on
-> If we do not fund this project, will your team still work on it? To what extent? *
-
-We will work on it also, but it will cost more times because we really need some outside supports now, to prove this project is meaningful and to encourage us working harder, for the next chapter of the internet.
-
++ Develop websites with our library for public service for free.
++ We are members of a large rust community in China, lots of rust developers in that community seems like to try our library.
```

Comment:
lots of rust developers in that community try to apply rust to front domain dev. Our library would be a good choice for them.

---

is better?

Reply:
y, we can commit to this one pr together, and merge it later, then we create a new pr for the last small modifications.

Reply:
OK